jetcd-core/src/test/java/io/etcd/jetcd/launcher/test/EtcdClusterUsingTest.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2016-2019 The jetcd authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.etcd.jetcd.launcher.test;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static org.junit.Assert.assertEquals;
+
+import io.etcd.jetcd.ByteSequence;
+import io.etcd.jetcd.Client;
+import io.etcd.jetcd.KV;
+import io.etcd.jetcd.KeyValue;
+import io.etcd.jetcd.kv.GetResponse;
+import io.etcd.jetcd.launcher.EtcdCluster;
+import io.etcd.jetcd.launcher.EtcdClusterFactory;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import org.junit.Test;
+
+/**
+ * Simple test illustrating how to use the {@link EtcdClusterFactory} with the jetcd client.
+ *
+ * <p>Tests typically would use the EtcdClusterResource JUnit rule instead of the EtcdClusterFactory direct.
+ *
+ * @author Michael Vorburger.ch
+ */
+public class EtcdClusterUsingTest {
+
+  @Test
+  public void testUseEtcd() throws Exception {
+    try (EtcdCluster etcd = EtcdClusterFactory.buildCluster(getClass().getSimpleName(), 3, false, false)) {
+      etcd.start();
+      try (Client client = Client.builder().endpoints(etcd.getClientEndpoints()).build()) {
+        try (KV kvClient = client.getKVClient()) {
+
+          ByteSequence key = ByteSequence.from("test_key", UTF_8);
```

Comment:
is it possible to import `import static io.etcd.jetcd.TestUtil.bytesOf`?
and then do `ByteSequence KEY = bytesOf("test_key");`?

Reply:
sure, done.